app/helpers/application_helper.rb

```diff
@@ -24,6 +24,10 @@ def bootstrap_class_for_flash(flash_type)
   end
 
   def display_date(date_time)
+    unless date_time.present?
+      date_time
+
+    end
     if date_time.is_a? String
```

Comment:
IMO, A switch case of instance checks for string, datetime? & defaulting to return date_time would be more readable than doing a `.present?` check followed by if/else.

Reply:
Will make it more readable